esf.tf

```diff
@@ -81,7 +82,7 @@ module "esf-lambda-function" {
     data.external.esf_lambda_loader
   ]
 
-  use_existing_cloudwatch_log_group = true
+  use_existing_cloudwatch_log_group = false
```

Comment:
NIT: it might be worth having this as a parameter at some point. If you want to modify an existing Lambda, this needs to be true.